jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHandler.java

```diff
@@ -156,7 +154,7 @@
 {
     public static final String GZIP = "gzip";
     public static final String DEFLATE = "deflate";
-    public static final int DEFAULT_MIN_GZIP_SIZE = 16;
+    public static final int BREAK_EVEN_GZIP_SIZE = 23;
```

Comment:
This is a public API, don't change this name.
Or at least don't delete the original name.